Software-Engineering.md

```diff
@@ -142,13 +142,13 @@ _We avoid distinguishing too much between the Junior Software Engineer and Softw
 - I have a deep understanding of Octopus products and our strategy
 - I identify significant problems, inhibitors and opportunities, and advocate for solutions that align our strategy and Engineering efforts
 - I collaborate with others to articulate solutions, positively influence and convince that we should fund their implementation
-- My contributions leave our Engineering team, and Octopus as a whole, in a stronger strategic position
+- My contributions are strategic and take our Engineering team, and Octopus as a whole, to the next level
 
 ### I am trusted to take care of a large, critical or complex strategic area or product
 
-- I am comfortable taking on a major challenge (e.g. re-write Octopus Server, or re-architect Octopus Cloud using microservices)
-- I identify team members with the necessary skills, work out how to fill gaps, and guide and influence that team (or teams) collaboratively
-- I use my Engineering experience to collaboratively steer teams toward good decisions along the way
+- I am comfortable taking on a major challenge that requires to build or re-architect an important part of Octopus ecosystem (e.g. Modularity, Octopus Cloud)
```

Comment:
Suggest phrasing such that new starters can understand, e.g.:

```suggestion
- I am comfortable taking on a major challenge that requires building or re-architecting an important part of the Octopus ecosystem (e.g. re-architecting Octopus Server to allow for plugin functionality, or creating a new Octopus Cloud platform)
```

Reply:
> Suggest phrasing such that new starters can understand, e.g.:

My goal is to be as specific here as possible.  Every company has a plugin system which does something else but there has been only one Modularity project at Octopus.  I would expect new people to learn about the project and understand why it was important for us.